SHELLY_MJS.md

```diff
@@ -1,3 +1,11 @@
+ble-events-handler.js: Handler script for BLE events
+===
+Example how to catch events from BLE devices and how to use them to trigger automations
```

Comment:
If this is to be used in tandem with ble-shelly-blu then we should add to the description